	Avoid some compiler warnings:
		remove unused variables
		add explicit cast for `-1' used as `uint32_t'

diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.85.orig/liblcdf/straccum.cc lcdf-typetools-2.85/liblcdf/straccum.cc
--- lcdf-typetools-2.85.orig/liblcdf/straccum.cc	2010-09-06 20:55:31.000000000 +0200
+++ lcdf-typetools-2.85/liblcdf/straccum.cc	2011-03-08 11:13:12.000000000 +0100
@@ -139,7 +139,6 @@
     else {
 	unsigned char *old_s = _s;
 	int old_len = _len;
-	int old_cap = _cap;
 
 	_s = 0;
 	_len = 0;
diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.85.orig/otftotfm/dvipsencoding.cc lcdf-typetools-2.85/otftotfm/dvipsencoding.cc
--- lcdf-typetools-2.85.orig/otftotfm/dvipsencoding.cc	2010-09-27 04:17:32.000000000 +0200
+++ lcdf-typetools-2.85/otftotfm/dvipsencoding.cc	2011-03-08 11:24:32.000000000 +0100
@@ -30,7 +30,7 @@
        GLYPHLIST_USEMAP = GLYPHLIST_ALTERNATIVE,
        U_EMPTYSLOT = 0xD801,
        U_ALTSELECTOR = 0xD802 };
-static HashMap<String, uint32_t> glyphlist(-1);
+static HashMap<String, uint32_t> glyphlist((uint32_t)-1);
 static Vector<uint32_t> glyphmap;
 static PermString::Initializer perm_initializer;
 PermString DvipsEncoding::dot_notdef(".notdef");
@@ -162,7 +162,6 @@
 		   // 16.Aug.2008: Some texnansx.enc have incorrect "Uni"
 		   // prefix, but we might as well understand it.
 		   || memcmp(component.data(), "Uni", 3) == 0)) {
-	int old_size = unis.size();
 	for (const char *s = component.begin() + 3;
 	     s < component.end();
 	     s += 4)
